scripts/compare-deposits/compare-deposits.go

```diff
@@ -154,11 +154,17 @@ func cliMain(c *cli.Context) error {
 				maxBlock = h
 			}
 		}
-		n := sort.Search(int(maxBlock-minBlock), func(i int) bool {
+		size := int64(maxBlock - minBlock)
+		blocks := make([]int64, size)
+		for i := int64(0); i < size; i++ {
+			blocks[i] = i
+		}
+
+		n, _ := slices.BinarySearchFunc(blocks, nextSupply, func(i int64, i2 int64) int {
```

Comment:
I think `slices.BinarySearchFunc` just doesn't fit here because we _do not_ have any real slice. So this can be reverted and `sort.Search` works much better for the case.

Reply:
Reverted